pennylane_cirq/simulator_device.py

```diff
@@ -132,7 +132,8 @@ def apply(self, operations, **kwargs):
 
         # We apply an identity gate to all wires, otherwise Cirq would ignore
         # wires that are not acted upon
-        self.circuit.append(cirq.IdentityGate(len(self.qubits))(*self.qubits))
+        for qb in self.qubits:
+            self.circuit.append(cirq.IdentityGate(1)(qb))
```

Comment:
So in the new approach we're looping over individual qubits, but before we passed all of the qubits to a single identity gate?

Was this done for clarity or was it broken?

Reply:
@trbromley It didn't work applying a single identity gate, so this was the simplest work-around I could think of. I noticed that it's actually done for the `SimulatorDevice` as well during the wires refactor, but I'm not sure about that change exactly.